pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role_policy/guide.fr-fr.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
+excerpt: "Comment associer un rôle Vsphere à une politique IAM"
+updated: 2024-05-24
+---
+
+# Table des matières
+1. [Objectif - Liens des guides IAM](#Objectif)
+2. [Prérequis pour commencer](#Prérequis)
+3. [Etape 1 - Créer une politique (optionnel)](#Instruction)
+4. [Etape 2 - Modifier une politique](#Etape2)
+5. [Etape 3 - Ajouter un rôle à une politique](#Etape3)
+4. [A suivre - Comment associer un utilisateur à une politique](#A suivre)
+5. [Fin - Aller plus loin](#Aller plus loin)
+
+---
+## Objectif
+
+**Dans ce guide, nous allons vous expliquer comment créer une politique globale et ajouter un rôle Vsphere**
+
+## Prérequis
+
+- Avoir un compte OVHcloud (voir guide :[Comment créer un compte OVHcloud](https://help.ovhcloud.com/csm/fr-account-create-ovhcloud-account?id=kb_article_view&sysparm_article=KB0043023)).
+- Disposer au préalable d'un ou plusieurs produits liés à ce compte OVHcloud (Hosted Private Cloud powered by VMware, Service Pack VMware etc..)
+- Savoir ["Créer et gérer des utilisateurs locaux sur un compte OVHcloud"](https://help.ovhcloud.com/csm/fr-account-managing-users?id=kb_article_view&sysparm_article=KB0043058).
+- Avoir une politique IAM [Comment créer une politiques IAM"](https://help.ovhcloud.com/csm/fr-customer-iam-policies-ui?id=kb_article_view&sysparm_article=KB0058730).
+
+## Instruction
+
+## Etape 1
+
+### Créer une politique ? (optionnel)
+
+### Accéder à IAM 
+
+1. Pour accéder au service IAM, suivez les indications de la capture suivante : `SUPPORT STANDARD > IDENTITES ET ACCES (IAM)`{.action}
```

Comment:
Too many numbers can lead to confusion.
Also, the text should contain all the necessary information. Screenshots are a + but shall not be considered as standalone explanations.
```suggestion
1Pour accéder au service IAM, suivez les indications de la capture suivante : `SUPPORT STANDARD > IDENTITES ET ACCES (IAM)`{.action}
```